libr/core/cmd_api.c

```diff
@@ -7,6 +7,8 @@
 #include <r_cmd.h>
 #include <r_util.h>
 
+#define MAX_CHILDREN_SHOW 7
```

Comment:
why seven? can you add a comment explaining the use of this?

Reply:
Random number... I had to set a limit somewhere. I will add a comment.